.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: Terraform Validation
+
+on: [ push ]
+
+jobs:
+  test:
+    name: Verify Terraform is valid for ${{ matrix.terraform_dir }}
+    runs-on: ubuntu-18.04
+    timeout-minutes: 5
+    container:
+      image: hashicorp/terraform:0.12.19
+    env:
+      AWS_DEFAULT_REGION: us-east-1
```

Comment:
Set AWS_DEFAULT_REGION as per hashicorp/terraform#21408 (comment) to not fail on implicit provider defaults.